### ISSUE TITLE
Messed up panel menu icons when .desktop file contains png larger than 32x32 pixels

### ISSUE THREAD

**Issue by azizLIGHT** [darktrojan/openwith#32]
Smaller than [32x32](http://i.imgur.com/bZFHmVy.png) pixels looks OK
But [48x48](http://i.imgur.com/E0ATmF2.png), [128x128](http://i.imgur.com/fmP8ALA.png), and [512x512](http://i.imgur.com/EzYYZT7.png) look bad. The larger the png, the bigger the panel menu gets.

Copy of .desktop file:
```
[Desktop Entry]
Name=Firefox Nightly
GenericName=Firefox Nightly Edition
Exec=/home/aziz/bin/firefox
Icon=/home/aziz/Downloads/HP-Firefox-icon-512.png
Terminal=false
Type=Application
Categories=Application;Network;X-Developer;
Comment=Firefox Nightly Edition Web Browser.
```

Icon and sizes were obtained from [here](http://www.iconarchive.com/show/hydropro-icons-by-media-design/HP-Firefox-icon.html)

I am on: 
* Ubuntu 14.04 x64
* Firefox 37.0.1

**Comment by darktrojan** [comment]
I need to have another look at this. I took out the code restricting icon sizes because it caused more trouble than it fixed. Hopefully now the Firefox themes have stabilized and actually agree on things.

In the meantime set the icon to a 16px icon, using `about:config`, like this:
```
extensions.openwith.manual.JdTtRGGc.icon = file:///opt/firefox-nightly/browser/chrome/icons/default/default16.png
```

**Comment by darktrojan** [comment]
This *should* be fixed in v5.9. Reopen if it isn't.
